src/debug/attach-test.js

```diff
@@ -15,7 +15,7 @@ describe('Attach test', () => {
         return (async () => {
             config = new AttachTest();
             DATA_ROOT = path.join(ROOT, config.workspaceRoot);
-            let cmdStr = 'cd dir&&cd src&&javac ./test/attachdebug.java&&java -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=1044 test.attachdebug';
+            let cmdStr = 'cd dir&&cd src&&javac -g ./test/attachdebug.java&&java -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=1044 test.attachdebug';
```

Comment:
`cd ${DATA_ROOT}/src && javac -g ./test/attachdebug.java && java -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=1044 test.attachdebug`

Reply:
user string template to replace 
https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/template_strings

Reply:
Got it.